developer/variables.boot.volume.pkrvars.hcl

```diff
@@ -0,0 +1,6 @@
+SUBNET_ID="0727-9447a2c3-efb8-44d2-a180-115531e0b0d3"
```

Comment:
Can we remove references int file and follow standard ones.

Reply:
revoked this change